### PR TITLE
widget design

### PR DESCRIPTION
**Pull request by Pasha8914**
# Pull request template

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] branch rebased on `master`
- [x] styleguide check `npm run validate`
- [x] good naming
- [x] keep simple
- [x] video or screenshot attached
- [x] testing instruction attached
- [x] check your PR once again

### What and how to test

yarn run dev:testnet-widget

compare designs between widget and https://swaponline.github.io

1 -  removed usdt-btc swap banner for widget
2 - removed some unused code
3 - edited  some widget styles and some logical conditions

### Video or screenshot

https://youtu.be/5fvqCRFQ0Nk

To test it locally run ```git fetch origin refs/pull/2590/merge:pr2590 && git checkout pr2590``` where 2590 is number of this pull request

### CODE REVIEW

File: shared/components/PreventMultiTabs/PreventMultiTabs.js [hunk 1]
```diff
@@ -4,58 +4,58 @@ import WidthContainer from 'components/layout/WidthContainer/WidthContainer'
 import { FormattedMessage } from 'react-intl'
 import Button from 'components/controls/Button/Button'
 import config from 'app-config'
-import { constants, localStorage } from "helpers"
+import { constants, localStorage } from 'helpers'
```
Comment (Pasha8914) [inline shared/components/PreventMultiTabs/PreventMultiTabs.js:7]: npm run validate

File: shared/pages/PartialClosure/PartialClosure.js [hunk 2]
```diff
@@ -1039,7 +1036,6 @@ export default class PartialClosure extends Component {
       estimatedFeeValues,
       haveAmount,
       customWallet,
-      destinationSelected,
```
Comment (Pasha8914) [inline shared/pages/PartialClosure/PartialClosure.js:1042]: unused

File: shared/pages/PartialClosure/PartialClosure.js [hunk 3]
```diff
@@ -1029,7 +1027,6 @@ export default class PartialClosure extends Component {
       isDeclinedOffer,
       isFetching,
       maxAmount,
-      customWalletUse,
```
Comment (Pasha8914) [inline shared/pages/PartialClosure/PartialClosure.js:1032]: unused

File: shared/pages/PartialClosure/PartialClosure.js [hunk 4]
```diff
@@ -1012,9 +1012,7 @@ export default class PartialClosure extends Component {
       currenciesData,
       tokensData,
       intl: { locale, formatMessage },
-      userEthAddress,
       isOnlyForm,
-      allOrders
```
Comment (Pasha8914) [inline shared/pages/PartialClosure/PartialClosure.js:1017]: unused

File: shared/pages/PartialClosure/PartialClosure.js [hunk 5]
```diff
@@ -1012,9 +1012,7 @@ export default class PartialClosure extends Component {
       currenciesData,
       tokensData,
       intl: { locale, formatMessage },
-      userEthAddress,
```
Comment (Pasha8914) [inline shared/pages/PartialClosure/PartialClosure.js:1015]: unused

File: shared/pages/Swap/FailControler/FailControler.js [hunk 6]
```diff
@@ -5,15 +5,7 @@ import cssModules from 'react-css-modules'
 
 import CopyToClipboard from 'react-copy-to-clipboard'
 import { FormattedMessage } from 'react-intl'
-import { BigNumber } from 'bignumber.js'
```
Comment (Pasha8914) [inline shared/pages/Swap/FailControler/FailControler.js:8]: unused

File: shared/pages/Wallet/Wallet.js [hunk 7]
```diff
@@ -402,21 +402,11 @@ export default class Wallet extends Component {
       activeView,
       infoAboutCurrency,
       exchangeForm,
-      editTitle,
-      walletTitle,
```
Comment (Pasha8914) [inline shared/pages/Wallet/Wallet.js:406]: unused

File: shared/pages/Wallet/components/BalanceForm/BalanceForm.js [hunk 8]
```diff
@@ -76,14 +76,13 @@ function BalanceForm({
                 ""
               )} */}
             </p>
-          ) : (
-            <p className="data-tut-all-balance">
+          ) : <p className="data-tut-all-balance">
```
Comment (Pasha8914) [inline shared/pages/Wallet/components/BalanceForm/BalanceForm.js:79]: couldn't defeat linter, npm run validate swear, had to do it

File: shared/pages/Wallet/components/BalanceForm/BalanceForm.js [hunk 9]
```diff
@@ -115,25 +114,19 @@ function BalanceForm({
         })}
       >
         <div styleName="yourBalanceBottom">
-          {showButtons ? (
```
Comment (Pasha8914) [inline shared/pages/Wallet/components/BalanceForm/BalanceForm.js:118]: couldn't defeat linter, npm run validate swear, had to do it

File: shared/pages/Wallet/components/WallerSlider/index.js [hunk 10]
```diff
@@ -7,14 +7,11 @@ import security from '../NotityBlock/images/security.svg'
 import styles from '../NotityBlock/NotifyBlock.scss'
 import NotifyBlock from '../NotityBlock/NotifyBock'
 import ContentLoader from '../../../../components/loaders/ContentLoader/ContentLoader'
-import config from 'app-config'
 
 import { FormattedMessage } from 'react-intl'
 import linksManager from '../../../../helpers/links'
 
 
-const isWidgetBuild = config && config.isWidget
```
Comment (Pasha8914) [inline shared/pages/Wallet/components/WallerSlider/index.js:16]: unused

File: shared/pages/Wallet/components/WallerSlider/index.js [hunk 11]
```diff
@@ -53,12 +50,15 @@ export default class WallerSlider extends Component {
   }
 
   getBanners = () => {
+    const { isWidgetBuild } = this.props
     try {
       return axios
         .get('https://noxon.wpmix.net/swapBanners/banners.php')
-        .then(result => {
+        .then(({ data }) => {
```
Comment (Pasha8914) [inline shared/pages/Wallet/components/WallerSlider/index.js:57]: remove usdt-btc banner for widget slider

File: shared/components/PreventMultiTabs/PreventMultiTabs.js [hunk 12]
```diff
@@ -4,58 +4,58 @@ import WidthContainer from 'components/layout/WidthContainer/WidthContainer'
...
+        {isWidgetBuild && (
+          <FormattedMessage
+            id="PreventMultiTabsWidgetBuild"
+            defaultMessage="Atomic Swap Widget supports only one active tab. Please reload this page to continue using this tab or close it"
+          />
+        )}
+        {!isWidgetBuild && (
+          <FormattedMessage
+            id="PreventMultiTabs12"
+            defaultMessage="Our service Supports only one active tab. Please reload this page to continue using this tab or close it"
```
Comment (noxonsu) [inline shared/components/PreventMultiTabs/PreventMultiTabs.js:56]: оставить только это, убрать " {!isWidgetBuild

File: shared/pages/Wallet/components/WallerSlider/index.js (hunk 11 above)
Comment (noxonsu) [reply shared/pages/Wallet/components/WallerSlider/index.js:57]: зачем? в виджете же тоже есть обменник, его надо оставить

File: shared/components/PreventMultiTabs/PreventMultiTabs.js (hunk 12 above)
Comment (shendel) [reply shared/components/PreventMultiTabs/PreventMultiTabs.js:56]: Там разница только в тексе. Есть упоминание swap.online , а в виджете - просто, "Наш сервис ...."

**Pull request merged**